docs/generator.md

```diff
@@ -39,6 +39,7 @@ minified               | boolean  | `false`         | Should the output be minif
 concise                | boolean  | `false`         | Set to `true` to reduce whitespace (but not as much as `opts.compact`)
 filename               | string   |                 | Used in warning messages
 jsonCompatibleStrings  | boolean  | `false`         | Set to true to run `jsesc` with "json": true to print "\u00A9" vs. "©";
+jsescOption            | string   |                 | Given some data, jsesc returns an ASCII-safe stringified representation of that data. Refer https://github.com/mathiasbynens/jsesc#api for the shape of `jsescOption` and more details.
```

Comment:
I think it's important to describe `jsescOption` within the context of @babel/generator, maybe something like:
 
```suggestion
jsescOption            | string   |                 | We leverage `jsesc` for getting ASCII-safe output when stringif-ying values. You can customize `jsesc` by [passing options](https://github.com/mathiasbynens/jsesc#api) to it.
```

?

Reply:
@existentialism Made the changes as suggested. PTAL. Thanks :)